test/e2e/cloud/imports.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloud
+
+import (
+	// ensure libs have a chance to perform initialization
+	_ "k8s.io/kubernetes/test/e2e/cloud/gcp"
```

Comment:
`init()` is an anti-pattern :[

Reply:
I just copy-pasted the comment, really this is about ensuring the tests are picked up by ginkgo, I can change the comment accordingly

Reply:
just noting in general about init().
i don't have improvement proposal for the time being.